lib/libutee/tee_api_objects.c

```diff
@@ -397,20 +397,43 @@ TEE_Result TEE_OpenPersistentObject(uint32_t storageID, void *objectID,
 				    uint32_t objectIDLen, uint32_t flags,
 				    TEE_ObjectHandle *object)
 {
-	if (storageID != TEE_STORAGE_PRIVATE)
-		return TEE_ERROR_ITEM_NOT_FOUND;
+	TEE_Result res;
 
-	if (objectID == NULL)
-		return TEE_ERROR_ITEM_NOT_FOUND;
+	if (storageID != TEE_STORAGE_PRIVATE) {
+		res = TEE_ERROR_ITEM_NOT_FOUND;
```

Comment:
Why not return this directly, there's nothing special for this at the end any way.